backend/src/openarchiefbeheer/destruction/tests/endpoints/test_co_reviews.py

```diff
@@ -0,0 +1,147 @@
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+                "list_feedback": "gh-497",
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_create_not_co_reviewer(self):
+        user = DestructionListAssigneeFactory.create()
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        destruction_list.assignees.add(user)
+
+        self.client.force_login(user.user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.post(
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+                "list_feedback": "gh-497",
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_create_not_assigned(self):
+        co_reviewer = DestructionListAssigneeFactory.create(role=ListRole.co_reviewer)
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        destruction_list.assignees.add(co_reviewer)
+        user = UserFactory(post__can_co_review_destruction=True)
+
+        self.client.force_login(user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.post(
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+                "list_feedback": "gh-497",
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_create_no_list_feedback(self):
+        co_reviewer = DestructionListAssigneeFactory.create(role=ListRole.co_reviewer)
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        destruction_list.assignees.add(co_reviewer)
+
+        self.client.force_login(co_reviewer.user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.post(
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_create(self):
+        co_reviewer = DestructionListAssigneeFactory.create(role=ListRole.co_reviewer)
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        destruction_list.assignees.add(co_reviewer)
+
+        self.client.force_login(co_reviewer.user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.post(
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+                "list_feedback": "gh-497",
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
Maybe good to check that the author field of the DestructionListCoReview is set properly based on the user that made the request?

Reply:
- [ ] Expand test